package.xml

```diff
@@ -9,10 +9,12 @@
     Wrapper around console_bridge, providing nothing but a dependency on console_bridge, on some systems.
     On others, it provides an ExternalProject build of console_bridge.
   </description>
-  <maintainer email="steven@osrfoundation.org">Steven! Ragnarök</maintainer>
+  <maintainer email="alejandro@osrfoundation.org">Alejandro Hernandez</maintainer>
   <license>Apache License 2.0</license>  <!-- the contents of this package are Apache 2.0 -->
   <license>BSD</license>  <!-- console_bridge is BSD -->
 
+  <author>Steven! Ragnarök</author>
```

Comment:
I do believe that the `author` tag should go after `url`; I think that will make the linters pass.